plugin-server/src/worker/ingestion/process-event.ts

```diff
@@ -210,7 +210,9 @@ export class EventsProcessor {
 
         const elementsChain = elements && elements.length ? elementsToString(elements) : ''
 
-        const groupProperties = await this.db.getGroupProperties(teamId, this.getGroupIdentifiers(properties))
+        const groupIds = this.getGroupIdentifiers(properties)
```

Comment:
Suggestion: Let's keep the naming consistent and use `groupIdentifiers` everywhere. groups have ids which is a completely separate concept and mixing the two makes the code harder to follow.